sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/parquet/ParquetScan.scala

```diff
@@ -40,6 +40,7 @@ case class ParquetScan(
     readDataSchema: StructType,
     readPartitionSchema: StructType,
     filters: Array[Filter],
+    pushedFilters: Array[Filter],
```

Comment:
The `pushedFilters` is added here just for end-to-end test. Not sure if this is appropriate.